tests/5.0/teams_loop/test_target_teams_loop_depend.c

```diff
@@ -0,0 +1,49 @@
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 1024
+#define INC 7
+#define MUL 11
+
+/**
+  This is a basic test to demonstrate how depend clause in various
+  combinations.
+*/
+int testDependAddToMulSync() {
+  int a[N], b[N], c[N];
+  int errors = 0;
+  // Data Inititalize
+  for (int i = 0; i < N; i++) {
+    a[i] = i;
+    b[i] = 0;
+    c[i] = 0;
+  }
+  // Execute on target
+#pragma omp target teams loop map(tofrom: a[0:N]) depend(out: a[0:N])
```

Comment:
Add `nowait` clauses to this and other target directives, and add `taskwait` directive after the last target construct. 
(Without `nowait` clauses, the generated target task will be an included task; thus, this test cannot check whether the `depend` clause works as intended.)

Reply:
added nowait as suggested...